anthos-bm-gcp-terraform/main.tf

```diff
@@ -101,18 +101,19 @@ module "instance_template" {
     module.enable_google_apis_secondary
   ]
   # fetched from previous module to explicitely express dependency
-  project_id           = module.enable_google_apis_secondary.project_id
-  region               = var.region           # --zone=${ZONE}
-  source_image         = var.image            # --image=ubuntu-2004-focal-v20210429
-  source_image_family  = var.image_family     # --image-family=ubuntu-2004-lts
-  source_image_project = var.image_project    # --image-project=ubuntu-os-cloud
-  machine_type         = var.machine_type     # --machine-type $MACHINE_TYPE
-  disk_size_gb         = var.boot_disk_size   # --boot-disk-size 200G
-  disk_type            = var.boot_disk_type   # --boot-disk-type pd-ssd
-  network              = var.network          # --network default
-  tags                 = var.tags             # --tags http-server,https-server
-  min_cpu_platform     = var.min_cpu_platform # --min-cpu-platform "Intel Haswell"
-  can_ip_forward       = true                 # --can-ip-forward
+  project_id                   = module.enable_google_apis_secondary.project_id
+  region                       = var.region           # --zone=${ZONE}
+  source_image                 = var.image            # --image=ubuntu-2004-focal-v20210429
+  source_image_family          = var.image_family     # --image-family=ubuntu-2004-lts
+  source_image_project         = var.image_project    # --image-project=ubuntu-os-cloud
+  machine_type                 = var.machine_type     # --machine-type $MACHINE_TYPE
+  disk_size_gb                 = var.boot_disk_size   # --boot-disk-size 200G
+  disk_type                    = var.boot_disk_type   # --boot-disk-type pd-ssd
+  network                      = var.network          # --network default
+  tags                         = var.tags             # --tags http-server,https-server
+  min_cpu_platform             = var.min_cpu_platform # --min-cpu-platform "Intel Haswell"
+  can_ip_forward               = true                 # --can-ip-forward
+  enable_nested_virtualization = true                 # --enable-nested-virtualization
```

Comment:
Can we set this as a variable in `variable.tf` and provide a default value?

Reply:
Yes